.travis.yml

```diff
@@ -49,6 +49,7 @@ before_install:
 
 install:
     - if [[ $deps != skip ]]; then COMPONENTS=$(find src/Symfony -mindepth 3 -type f -name phpunit.xml.dist -printf '%h\n'); fi;
+    - COMPONENTS=src/Symfony/Component/Process
```

Comment:
This should be removed before merging (just adding the comment here to make sure it is not forgotten as github will hide it when it is done)